server/server_test.go

```diff
@@ -342,6 +345,59 @@ func TestDataPutV1IfNoneMatch(t *testing.T) {
 	}
 }
 
+func TestDataGetWatch(t *testing.T) {
+	f := newFixture(t)
+
+	type trw struct {
+		tr   tr
+		wait chan struct{}
+	}
+
+	// Test watching /data.
+	exp := strings.Join([]string{
+		"HTTP/1.1 200 OK\nContent-Type: application/json\nTransfer-Encoding: chunked\n\n57",
+		`{"result":[{"expressions":[{"value":{},"text":"data","location":{"row":1,"col":1}}]}]}
```

Comment:
I would like to make the response format for the /data/...?watch API calls match the non-?watch API calls.

This would mean replacing the result value with the expression value that's currently returned. So this test case would look like:

```
HTTP/1.1 200 OK
Content-Type: application/json
Transfer-Encoding: chunked

{"result": {}}
{"result": {"x": {"a":1,"b":2}}}
...
```